.circleci/config.yml

```
@@ -234,7 +232,9 @@ jobs:
     parallelism: 2
     working_directory: /home/qa/code
     docker:
-      - image: lecaoquochung/docker-images:puppeteer-v1.1.0-build.1
+      # - image: lecaoquochung/docker-images:puppeteer-v1.1.0-build.1
+      # - image: ghcr.io/lecaoquochung/docker-images:puppeteer-latest-build # Dev
+      - image: ghcr.io/lecaoquochung/docker-images:puppeteer-latest-v3.3.0 # Stable
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)